### ISSUE TITLE
Botania Mana Pool consistently crashes game when loading chunks with them in it

### ISSUE THREAD

**Issue by thederpmachine** [xJon/The-1.12.2-Pack#277]
**Basic details**
1. Are you using a legitimate launcher and Minecraft account:
yes.
2. Are you using a computer with 8GB or more of RAM? ([How to check](https://www.wikihow.com/Check-Computer-RAM)):
yes.
3. Are you using Java 8, 64-bit? ([How to check](https://www.wikihow.com/Determine-Java-Version)):
yes.

**Information required**
1. Is the modpack crashing: yes
2. Crash/latest log *link* ([Where to find](https://github.com/xJon/The-1.12.2-Pack/blob/master/log_location.md#finding-modpack-installation-folder)): https://paste.dimdev.org/ekihozaqen.mccrash
3. Is Optifine installed or any other additional mods: no

**Describe the issue**

Basically, the mana pool from botania, when placed down and used, consistently crashes the game of anyone who has to re-load chunks with the pool in them, for example when coming back from the nether. I'm almost certain this block is causing the issue, because when it was removed from the world everything was fine.

**Comment by xJon** [comment]
https://github.com/xJon/The-1.12.2-Pack/issues/13
Either cover your Botania stuff so VoxelMap doesn't try to render them, or replace VoxelMap with another minimap mod.
